BinaryEncoding.md

```diff
@@ -65,20 +65,36 @@ Yes:
 
 ## Global structure
 
-* A module contains:
-  * a header followed by
-  * a table (sorted by offset) containing, for each section, its type and offset (within the module), followed by
-  * a sequence of sections.
+* A module contains (in this order):
```

Comment:
Can you use the different styles of bullet lists below, so it's easier to read the text. E.g.:
* A
  - Deeply
    + Nested
  - List
  - Is
* Easier
* To
  - Read
    + With
    + Different
    + Bullets